options/options.go

```diff
@@ -82,7 +83,13 @@ type TerragruntOptions struct {
 func NewTerragruntOptions(terragruntConfigPath string) *TerragruntOptions {
 	workingDir := filepath.Dir(terragruntConfigPath)
 
-	downloadDir := filepath.Join(os.TempDir(), "terragrunt")
+	user, err := user.Current()
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "error: %v\n", err)
```

Comment:
Just a few lines below, we call `util.CreateLogger()`. Move that up here and use it to do the logging instead of `fmt.Fprintf`.